api/src/repositories/submission-repository.ts

```diff
@@ -223,7 +240,7 @@ export class SubmissionRepository extends BaseRepository {
    */
   async updateSubmissionRecordInputKey(
     submissionId: number,
-    inputKey: IInsertSubmissionRecord['input_key']
+    inputKey: string
```

Comment:
`IInsertSubmissionRecord` will lose most of its params due to DB restructure; We will need a new `IInsertSubmissionMetadataRecord` interface and corresponding repo method